model/src/test/kotlin/licenses/ResolvedLicenseTest.kt

```diff
@@ -19,41 +19,74 @@
 
 package org.ossreviewtoolkit.model.licenses
 
-import io.kotest.core.spec.style.StringSpec
+import io.kotest.core.spec.style.WordSpec
 import io.kotest.matchers.collections.shouldContainExactlyInAnyOrder
 import io.kotest.matchers.collections.shouldHaveSize
 import io.kotest.matchers.shouldBe
 
 import org.ossreviewtoolkit.model.TextLocation
 
-class ResolvedLicenseTest : StringSpec({
-    "Copyright findings with the same statement across different files should be listed in the result" {
-        val originalFindings = listOf(
-            ResolvedCopyrightFinding(
-                statement = "Copyright (C) 2022 The ORT Project Authors",
-                location = TextLocation(
-                    path = "/path/to/file/A",
-                    line = 2
+class ResolvedLicenseTest : WordSpec({
+    "Copyright findings across different files" should {
+        "be consolidated without processing if their statements are exactly the same" {
+            val originalFindings = listOf(
+                ResolvedCopyrightFinding(
+                    statement = "Copyright (C) 2022 The ORT Project Authors",
+                    location = TextLocation(
+                        path = "/path/to/file/A",
+                        line = 2
+                    ),
+                    matchingPathExcludes = emptyList()
                 ),
-                matchingPathExcludes = emptyList()
-            ),
-            ResolvedCopyrightFinding(
-                statement = "Copyright (C) 2022 The ORT Project Authors",
-                location = TextLocation(
-                    path = "/path/to/file/B",
-                    line = 2
+                ResolvedCopyrightFinding(
+                    statement = "Copyright (C) 2022 The ORT Project Authors",
+                    location = TextLocation(
+                        path = "/path/to/file/B",
+                        line = 2
+                    ),
+                    matchingPathExcludes = emptyList()
                 ),
-                matchingPathExcludes = emptyList()
-            ),
-        )
+            )
 
-        val resolvedCopyrights = originalFindings.toResolvedCopyrights(process = false)
+            val resolvedCopyrights = originalFindings.toResolvedCopyrights(process = false)
```

Comment:
If I didn't miss anything these two test cases only differ in terms of the boolean value for `process`, e.g. `true` vs. `false`. While it make sense to test one thing at a time, I believe in this case it would make sense to merge these tests like: 
```
should "be consolidated with and without processing if their statements are exactly the same" {
...

listOf(true, false).forall { process ->
  // assert here
}
```
What do you think?

Reply:
> If I didn't miss anything these two test cases only differ in terms of the boolean value for `process`

No, the input `statement` also differs see the `// Note the "." at the end.` comment. Thus I'd prefer to keep these test cases separate.